### PR TITLE
Plone theme package structure

### PR DESCRIPTION
**Pull request by macagua**
Hi @tkimnguyen please, can you check these improvements and if all are Ok, can merged it?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+License
+=======
+
+The project is licensed under the GPLv2.
+
+Credits
+-------
+
+- T\. Kim Nguyen (nguyen at gmail dot com).
```
Comment (tkimnguyen) [inline README.rst:121]: that email address is incorrect; probably best to leave my contact info as `https://github.com/tkimnguyen`

Comment (tkimnguyen) [reply README.rst:121]: or https://twitter.com/tkimnguyen as below

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Features
+========
+
+- It's an installable Plone Theme package.
+- After installation from Add-ons controlpanel, this theme is enabled automatically.
+- It's have support for clean uninstallation.
+- Also it's a simple Diazo package (Zip file).
+
+
+Installation
```
Comment (tkimnguyen) [inline README.rst:41]: Might make sense to remove this from here and put it in INSTALL.txt

**Review by tkimnguyen — APPROVED** [review]
Thanks Leonardo! I'm going to go ahead and merge and then fix my contact info but if you could still move the installation instructions out of README.rst and into INSTALL.txt that would be great

**Comment by macagua** [comment]
@tkimnguyen I sorry for the delay in response!, thanks for your improvements and merge it!

Also I transfers my forked repository into the "collective" organization to share this theme with all Plone guys!!! 

https://github.com/collective/plonetheme.chalkboard

**Comment by tkimnguyen** [comment]
Thanks for doing that @macagua

**Pull request merged**